tox.ini

```diff
@@ -1,5 +1,6 @@
 [tox]
-envlist = lint,py27,py34,doc,sphinx
+envlist = lint,py27,py34,py35,pep8py2,pep8py3,doc,sphinx
```

Comment:
One target tests compatibility with Python 2, the other runs flake8 under Python 3.

see line 28 and 38 in tox.in